### PR TITLE
Reuse WebDriver in Faces Selenium Tests

### LINKED ISSUE #26234: Faces Selenium Improvements
Kyle Aure recommend the following changes: 

```Using @Rule here will make it so the chrome container is started before each test, and stopped after each test. This is bad for performance, unless this is necessary I'd recommend switching to use @ClassRule```

```If possible try to reduce the number of times a container is started/stopped by switching to @ClassRule or by starting/stopping the container in the FATSuite.class instead.```

This requires clearing the session for each test. I will follow up later on. 

Changes should apply to existing FAT buckets where selenium updates applied: 2.2, 2.3 and 2.0.

### PR DESCRIPTION
**Pull request by volosied**
Use one driver and clear cookies in between tests.

fixes #26234

### CODE REVIEW

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat

**Comment by volosied** [comment]
Still failing with this error:
```
org.testcontainers.containers.ContainerLaunchException: Container startup failed for image docker-eu-public.artifactory.swg-devops.com/wasliberty-docker-remote/selenium/standalone-chrome:4.8.3
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:349)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)
	at org.testcontainers.containers.GenericContainer.starting(GenericContainer.java:1131)
	at org.testcontainers.containers.FailureDetectingExternalResource$1.evaluate(FailureDetectingExternalResource.java:28)
	at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:363)
	at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:177)
	at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:145)
Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)
Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:553)
	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344)
	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
Caused by: com.github.dockerjava.api.exception.BadRequestException: Status 400: {"message":"Duplicate mount point: /dev/shm"}

	at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.execute(DefaultInvocationBuilder.java:237)
	at org.testcontainers.shaded.com.github.dockerjava.core.DefaultInvocationBuilder.post(DefaultInvocationBuilder.java:124)
	at org.testcontainers.shaded.com.github.dockerjava.core.exec.CreateContainerCmdExec.execute(CreateContainerCmdExec.java:37)
	at org.testcontainers.shaded.com.github.dockerjava.core.exec.CreateContainerCmdExec.execute(CreateContainerCmdExec.java:13)
	at org.testcontainers.shaded.com.github.dockerjava.core.exec.AbstrSyncDockerCmdExec.exec(AbstrSyncDockerCmdExec.java:21)
	at org.testcontainers.shaded.com.github.dockerjava.core.command.AbstrDockerCmd.exec(AbstrDockerCmd.java:35)
	at org.testcontainers.shaded.com.github.dockerjava.core.command.CreateContainerCmdImpl.exec(CreateContainerCmdImpl.java:608)
	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:427)
```

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

File: dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22ClientWindowTests.java [hunk 1]
```diff
@@ -92,9 +96,10 @@ public static void tearDown() throws Exception {
         }
     }
 
-    @Before
-    public void setupTest() {
-        driver = new CustomDriver(new RemoteWebDriver(chrome.getSeleniumAddress(), new ChromeOptions().setAcceptInsecureCerts(true)));
+    @After
+    public void after()
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22ClientWindowTests.java]: Rename

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsf.2.3_fat,com.ibm.ws.jsf.2.2_fat

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#libby

**Pull request merged**